functions/src/api/v1/users/getUser.ts

```diff
@@ -0,0 +1,57 @@
+import { Request, Response } from 'express'
+import { ForbiddenError } from '../../errors/HttpError'
+import { getUserDb } from '../../../db/getUserDb'
+
+/**
+ * @swagger
+ * /v1/users/{userId}:
+ *   get:
+ *     description: Get user details, subscription and quota status. This will not send any payment details or API keys.
+ *     tags: ['users']
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: User details
+ *         content:
+ *           application/json:
+ *             example: {
+ *               id: 'vXIUuaGkH4kukbwRH5cU',
+ *               updatedAt: 1605969562,
+ *               subscription: {
+ *                   isActive: true,
+ *                   isQuotaReached: false
+ *               },
+ *               sentEmails: 1,
+ *               sentSMSs: 2
+ *             }
+ *             schema:
+ *               type: object
+ *               $ref: '#/components/schemas/User'
+ *       401:
+ *         description: missing authorization bearer token
+ *       403:
+ *         description: authorization header present but not valid
+ *       412:
+ *         description: authorization header present but not formatted correctly
+ */
+export const getUser = async (req: Request, res: Response) => {
+    const currentUid = res.locals.uid
+    const requestedUserId = req.params.userId
+
+    if (requestedUserId !== currentUid) {
+        throw new ForbiddenError('Not authorized to get this resource')
```

Comment:
we should discuss again about error translations. If we put those in the frontend side (as I'd suggest), we'd probably need to insert a `string` error code to uniquely identify a translation

Reply:
ForbiddenError result in a 403, which make it easy to have the error on the frontend only.

Reply:
Yes the front receives the error code, what I meant is to receive a string ID to find the correct translation on front-end side

Reply:
open an issue! :D

Reply:
#331